sql/firefox_accounts_derived/fxa_log_auth_events_v1/query.sql

```diff
@@ -0,0 +1,35 @@
+SELECT
+  jsonPayload.fields.event,
+  jsonPayload.fields.user_id,
+  jsonPayload.fields.flow_id,
+  jsonPayload.fields.entrypoint,
+  jsonPayload.fields.device_id,
+  jsonPayload.fields.service,
+  timestamp,
+  jsonPayload.fields.useragent,
+  jsonPayload.fields.os_version,
+  jsonPayload.fields.utm_source,
+  jsonPayload.fields.utm_campaign,
+  jsonPayload.fields.utm_content,
+  jsonPayload.fields.utm_medium,
+  jsonPayload.fields.utm_term
+FROM
+  `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_auth_20*`
+WHERE
+  NOT (
+    REGEXP_CONTAINS(jsonPayload.fields.event, r"^email\.(\w+)\.bounced$")
+    OR REGEXP_CONTAINS(jsonPayload.fields.event, r"^email\.(\w+)\.sent$")
+    OR REGEXP_CONTAINS(jsonPayload.fields.event, r"^flow\.complete\.(\w+)$")
+  )
+  AND jsonPayload.fields.event NOT IN (
+    'account.confirmed',
+    'account.login',
+    'account.login.blocked',
+    'account.login.confirmedUnblockCode',
+    'account.reset',
+    'account.signed',
+    'account.created',
+    'account.verified',
+    'sms.installFirefox.sent'
+  )
```

Comment:
Kimmy - can you comment on why we exclude these patterns?

Reply:
Similar to the regexp mappings, these are explicit matches that are excluded from the table to reduce redundancy in what's stored here and in Amplitude.